src/test/unit/controller/RespondentApplicationDetailsController.test.ts

```diff
@@ -102,6 +103,21 @@ describe('Respondent application details controller', () => {
       expect(response.render).toHaveBeenCalledWith(TranslationKeys.RESPONDENT_APPLICATION_DETAILS, expect.anything());
       expect(request.session.userCase.genericTseApplicationCollection.at(0).value.applicationState).toBe('inProgress');
     });
+
+    it("doesn't change to inProgress when viewing request for information from claimant", async () => {
+      const userCase: Partial<CaseWithId> = {
+        genericTseApplicationCollection: [clone(mockRespAppWithRespRequstForClaimantInfo)],
+      };
+      const response = mockResponse();
+      const request = mockRequestWithTranslation({ userCase }, respondentApplicationDetailsRaw);
+
+      await new RespondentApplicationDetailsController().get(request, response);
+
+      expect(response.render).toHaveBeenCalledWith(TranslationKeys.RESPONDENT_APPLICATION_DETAILS, expect.anything());
+      expect(request.session.userCase.genericTseApplicationCollection.at(0).value.applicationState).toBe(
+        'notStartedYet'
```

Comment:
Can variables like notStartedYet be assigned as constants so can be shared between implementation and test?  otherwise they will be all over the place and difficult to maintain.

Reply:
Yeah, I think it's a bit subjective as [some say not to use production constants in test](https://dev.to/scottshipp/don-t-use-non-test-constants-in-unit-tests-3ej0#:~:text=Every%20developer%20wants%20to%20promote,maintainable%20and%20more%20correct%20code.) If you feel particularly strongly then yeah I'll change.